config/categories.yaml

```diff
@@ -40,9 +40,9 @@ pillars:
         db_value: 'Marcomms & Research'
         slug: 'marcomms-research'
 
-      - name: 'Travel'
-        db_value: 'Travel'
-        slug: 'travel'
+      - name: 'Office and Travel'
+        db_value: 'Office and Travel'
+        slug: 'office-and-travel'
```

Comment:
This hasn't followed the previous convention in the file, to leave out the 'and' in the URL

Reply:
ah right, because this was an 'and' and not '&' I thought it would be okay. I even checked another example 'general-common-goods-and-services' this also had and in the slug. I can change this if you like in another PR?